### PR TITLE
Add argument to select launching teleop

### PR DESCRIPTION
**Pull request by 708yamaguchi**
In this PR, I add argument to select launching teleop in `fetch.launch`.

I need this argument because I want to launch our original teleop program in other launch file.

Cc @knorth55

### CODE REVIEW

**Comment by knorth55** [comment]
We need this PR to customize our fetch teleoperation PS3joystick command.
We want to set similar joystick command as PR2.

**Review by moriarty — COMMENTED** [review]
Seems reasonable.

@erelson @cjds any thoughts before merging this?

**Review by cjds — APPROVED** [review]
LGTM good idea

File: fetch_bringup/launch/fetch.launch
Comment (erelson) [inline fetch_bringup/launch/fetch.launch:5]: Let's add these changes to freight.launch in freight_bringup, as well, to keep the files similar.  Thanks!

**Review by erelson — CHANGES_REQUESTED** [review]

**Comment by erelson** [comment]
Seems fine, though my own typical approach would just be to comment out the teleop portion of /etc/ros/$distro/robot.launch

Note that in order for this change to take effect on your current robot(s), you must manually update your robot.launch file!  The file you've modified is only used at install time when the fetch-melodic-config package creates /etc/ros/melodic/robot.launch by copying it.

**Comment by knorth55** [comment]
>Seems fine, though my own typical approach would just be to comment out the teleop portion of /etc/ros/$distro/robot.launch

> Note that in order for this change to take effect on your current robot(s), you must manually update your robot.launch file! The file you've modified is only used at install time when the fetch-melodic-config package creates /etc/ros/melodic/robot.launch by copying it.

Thank you for your information.
We already change `/etc/ros/indigo/robot.launch` for calibration data.

**Comment by moriarty** [comment]
> Seems fine, though my own typical approach would just be to comment out the teleop portion of /etc/ros/$distro/robot.launch
> 
> Note that in order for this change to take effect on your current robot(s), you must manually update your robot.launch file! The file you've modified is only used at install time when the fetch-melodic-config package creates /etc/ros/melodic/robot.launch by copying it.

I think commenting out code in `/etc` should be avoided. I think we should have this PR, and open a related issue for looking into `fetch-melodic-config` to use a symlink or something similar so that this change could become effective.

If you don’t want to add the frieght change, we can open a new issue for that and merge this PR as is.

File: fetch_bringup/launch/fetch.launch
Comment (708yamaguchi) [reply fetch_bringup/launch/fetch.launch:5]: I pushed an additional commit.
Thanks!

**Comment by 708yamaguchi** [comment]
Thank you for your review.
I pushed an additional commit and added freight change.

I also created a PR (https://github.com/fetchrobotics/fetch_robots/pull/41 ) for indigo.
Could you merge this if there is no problem?

**Pull request merged**
